TS3Client/TsDnsResolver.cs

```diff
@@ -261,7 +261,7 @@ private static string ResolveNickname(string nickname)
 				return null;
 			}
 
-			return splits[0];
+			return splits[0]; // TODO: Maybe return all and try the next if one fails?
```

Comment:
That would maybe be nice but the current design structure does not allow for this.
Also iirc the dns protocol specifies that every ip should be valid, and the client should coose one.

Reply:
I get that... Maybe just keep it as todo for some far future